layouts/shortcodes/cncf-landscape.html

```diff
@@ -53,7 +53,19 @@
   window.addEventListener("hashchange", hashChangeHandler, false);
   /* handle initially-set fragment */
   hashChangeHandler();
+
+  function checkUrl() {
+    let windowLocation = window.location.pathname;
+    var urlName = 'kubernetes-training-partner';
+    if (windowLocation == '/training/') {
```

Comment:
also this...
maybe at the end the shortcode can be called like this:
```
{{< cncf-landscape helpers=true category="kubernetes-training-partner" check-path="/training/">}}
```
and if `checkUrl` is actually only needs to be done in certain path, you could just avoid making this function and instead do something like this:

```js
{{ $checkPath := .Get "check-path" }}
{{ if $checkPath }}
if (window.location.pathname === "{{ $checkPath }}") {
  var landscapeElement = document.getElementById("landscape");
  landscapeElement.src = "https://landscape.cncf.io/category="+"{{ .Get "category" | default "some-default-cat" }}"+"&format=card-mode&grouping=category&embed=yes";
}
{{ end }}
```

Reply:
hmm... my confusion is do we really need to check whether we are in `/training/` to run this?

Reply:
> ```
> {{< cncf-landscape helpers=true category="kubernetes-training-partner" check-path="/training/">}}
> ```

Along those lines, yes. I'd had in mind something more like:
```
{{< cncf-landscape helpers=false category="kubernetes-training-partner" >}}
```

Reply:
It's even simpler then, no need to check for the path (which I have doubt to be required in this case) 😄